meta-mender-core/classes/mender-maybe-setup.bbclass

```diff
@@ -89,6 +89,9 @@ python() {
 
         # Enable the testing/* layers and functionality
         'mender-testing-enabled',
+
+        # Enable UEFI Capsule artifact generation
+        'mender-image-capsule',
```

Comment:
Should it be `mender-image-uefi-capsule`, for clarity? :thinking:

Reply:
done